.dockerignore

```diff
@@ -6,5 +6,8 @@
 Dockerfile
 Dockerfile.*
 heroku.yml
-libs/fixtures/data
```

Comment:
I think the recent changes for the move to vitest have made it so there's a transitive dependency on the fixture data somewhere - had to restore this. Will track down later and see if it's easy to pull out of the dependency chain